setup.py

```diff
@@ -19,7 +19,7 @@
     'codecov',
     'crds',
     'cryptography',
-    'django<=3.1.7',
+    'django<=3.2.5',
```

Comment:
Were there problems with versions less than `3.2.5`?

Reply:
I don't remember changing this myself, if I did it was not important 'cause I don't remember :P  so we can use the latest